integration_tests/runtests.py

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 import sys
+import os
```

Comment:
Have you looked into using https://pypi.org/project/pytest-repeat/ instead of modifying this script?

Reply:
no, I am not familiar with Python yet, thanks for the suggestion @revans2

Reply:
`pytest-repeat` works per se. It's very slow to start for larger repetition counts such as 1000. Interestingly, the way it inflates the test series from 1 2 3 to let us say count=3:  1 1 1 2 2 2 3 3 3 does not reproduce the particular problem as fast. 

But this peculiarity does not mean we have incorporate the script change given the functionality of pytest-repeat is more than sufficient in general, I can just keep it on a branch for reference.